key-manager/src/main/scala/org/bitcoins/keymanager/EncryptedMnemonic.scala

```diff
@@ -1,32 +1,60 @@
 package org.bitcoins.keymanager
 
-import java.time.Instant
-
 import org.bitcoins.core.compat.CompatEither
 import org.bitcoins.core.crypto._
 import org.bitcoins.crypto.{AesCrypt, AesEncryptedData, AesPassword, AesSalt}
 import scodec.bits.ByteVector
 
+import java.time.Instant
 import scala.util.{Failure, Success, Try}
 
-sealed trait MnemonicState {
+sealed trait SeedState {
```

Comment:
Do we want to think about using `extends MaskedToString` ? There can be a lot of sensitive information in these `SeedState`.

Reply:
These should be fine, since the sensitive information already extend `MaskedToString`